Playback stalls with unmuxed content when discontinuities are encountered

### What version of Hls.js are you using?

v0.14.5 

### What browser and OS are you using?

Google Chrome, Firefox (latest release). macOs High Sierra : 10.13.6

### Test stream:
<!-- If possible, please provide a test stream or page -->
<!-- You can paste your stream into the demo and provide the permalink here -->

Permalink: https://hls-js.netlify.app/demo/?src=https%3A%2F%2Fe5d03d25d9c84378a3a51f816578f84a.mediatailor.us-west-2.amazonaws.com%2Fv1%2Fmaster%2Faa2ad754e5dbe391bea5e8bf3615878ac1e0a803%2Fhlsjs_repro%2Findex.m3u8&demoConfig=eyJlbmFibGVTdHJlYW1pbmciOnRydWUsImF1dG9SZWNvdmVyRXJyb3IiOnRydWUsImR1bXBmTVA0IjpmYWxzZSwibGV2ZWxDYXBwaW5nIjotMSwibGltaXRNZXRyaWNzIjotMX0=

Stream: https://e5d03d25d9c84378a3a51f816578f84a.mediatailor.us-west-2.amazonaws.com/v1/master/aa2ad754e5dbe391bea5e8bf3615878ac1e0a803/hlsjs_repro/index.m3u8

### Checklist
<!-- Replace [ ] with [x] to check off the list -->
- [x] The issue observed is not already reported by searching on Github under https://github.com/video-dev/hls.js/issues
- [x] The issue occurs in the stable client on https://hls-js.netlify.com/demo and not just on my page
<!-- The stable client is built from the latest release -->
- [x] The issue occurs in the latest client on https://hls-js-dev.netlify.com/demo and not just on my page
<!-- The latest client is built from the head of the master branch -->
- [x] The stream has correct Access-Control-Allow-Origin headers (CORS)
- [x] There are no network errors such as 404s in the browser console when trying to play the stream

### Steps to reproduce
1. Please provide clear steps to reproduce your problem
2. If the bug is intermittent, give a rough frequency

Playback the endpoint listed above, we've been able to reproduce each time. 
At or around the transition to / from content to ad breaks, we see playback stalling for a long period of time.
Ad breaks are identified by EXT-X-DISCONTINUITY markers before and after the ad.

### Expected behavior
*What you expected to happen*

Playback should be stable through ad breaks.

Note - we only see this issue with unmuxed content, where requests are being made for both audio and video playlists. The issue seems to occur when HLS.js requests an audio playlist and sees a new discontinuity, but hasn't yet seen the corresponding discontinuity in the video playlist. The first sign of the issue in the console is the following:

```
[warn] > Waiting fragment CC (1) does not match video track CC (0)
```
followed by the bufferStalledError (logs attached below)

In past releases, we've added a line to audio-stream-controller.js to remove the fragment from the fragmentTracker so that it's re-requested, but the behavior has changed a bit in recent versions (we used to see the `Waiting fragment CC` message repeated in the console output). 

With muxed A/V, we see steady playback as expected, with smooth transitions in and out of discontinuities. 

### Actual behavior
*What actually happened*

Playback stalls when transitioning into or out of discontinuities. 

### Console output
```
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2757 ms
logger.js:40 [log] > level 0 loaded [795,815],duration:60
logger.js:40 [log] > live playlist sliding:3.000
logger.js:40 [log] > Updating Media Source duration to 63.000
logger.js:40 [log] > Loading 815 of [795 ,815],level 0, currentTime:53.135,bufferEnd:59.967
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > Loaded 815 of [795 ,815],level 0
logger.js:40 [log] > Parsing 815 of [795 ,815],level 0, cc 0
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > Unknown stream type:134
logger.js:40 [log] > Parsed video,PTS:[60.000,63.000],DTS:[59.967/62.967],nb:90,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,62.967]
logger.js:40 [log] > latency/loading/parsing/append/kbps:127/85/12/3/13439
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > track 0 loaded [795,815],duration:60.01300000000003
logger.js:40 [log] > live audio playlist sliding:3.009
logger.js:40 [log] > Loading 815, cc: 0 of [795 ,815],track 0, currentTime:53.37749,bufferEnd:60.014
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > Demuxing 815 of [795 ,815],track 0
logger.js:40 [log] > parsed audio,PTS:[60.014,63.022],DTS:[60.014/63.022],nb:141
logger.js:40 [log] > audio stream:PARSING->PARSED
logger.js:40 [log] > audio buffered : [0.001,3.009][51.011,63.022]
logger.js:40 [log] > audio stream:PARSED->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2759 ms
logger.js:40 [log] > level 0 loaded [796,816],duration:60
logger.js:40 [log] > live playlist sliding:6.000
logger.js:40 [log] > Updating Media Source duration to 66.000
logger.js:40 [log] > Loading 816 of [796 ,816],level 0, currentTime:56.000,bufferEnd:62.967
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 816 of [796 ,816],level 0
logger.js:40 [log] > Parsing 816 of [796 ,816],level 0, cc 0
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > Unknown stream type:134
logger.js:40 [log] > Parsed video,PTS:[63.000,66.000],DTS:[62.967/65.967],nb:90,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,65.967]
logger.js:40 [log] > latency/loading/parsing/append/kbps:123/86/20/13/14769
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [796,816],duration:59.99200000000002
logger.js:40 [log] > live audio playlist sliding:6.017
logger.js:40 [log] > Loading 816, cc: 0 of [796 ,816],track 0, currentTime:56.475262,bufferEnd:63.022
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > Demuxing 816 of [796 ,816],track 0
logger.js:40 [log] > parsed audio,PTS:[63.022,66.008],DTS:[63.022/66.008],nb:140
logger.js:40 [log] > audio stream:PARSING->PARSED
logger.js:40 [log] > audio buffered : [0.001,3.009][51.011,66.008]
logger.js:40 [log] > audio stream:PARSED->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2779 ms
logger.js:40 [log] > level 0 loaded [797,817],duration:60
logger.js:40 [log] > live playlist sliding:9.000
logger.js:40 [log] > Updating Media Source duration to 69.000
logger.js:40 [log] > Loading 817 of [797 ,817],level 0, currentTime:58.848,bufferEnd:65.967
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 817 of [797 ,817],level 0
logger.js:40 [log] > Parsing 817 of [797 ,817],level 0, cc 0
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > Unknown stream type:134
logger.js:40 [log] > Parsed video,PTS:[66.000,69.000],DTS:[65.967/68.967],nb:90,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,68.967]
logger.js:40 [log] > latency/loading/parsing/append/kbps:137/80/12/3/12829
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [797,817],duration:60.01300000000002
logger.js:40 [log] > live audio playlist sliding:9.004
logger.js:40 [log] > Loading 817, cc: 0 of [797 ,817],track 0, currentTime:59.584085,bufferEnd:66.008
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > Demuxing 817 of [797 ,817],track 0
logger.js:40 [log] > parsed audio,PTS:[66.008,69.016],DTS:[66.008/69.016],nb:141
logger.js:40 [log] > audio stream:PARSING->PARSED
logger.js:40 [log] > audio buffered : [0.001,3.009][51.011,69.016]
logger.js:40 [log] > audio stream:PARSED->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2771 ms
logger.js:40 [log] > level 0 loaded [798,818],duration:60
logger.js:40 [log] > live playlist sliding:12.000
logger.js:40 [log] > Updating Media Source duration to 72.000
logger.js:40 [log] > Loading 818 of [798 ,818],level 0, currentTime:61.724,bufferEnd:68.967
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 818 of [798 ,818],level 0
logger.js:40 [log] > Parsing 818 of [798 ,818],level 0, cc 0
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > Unknown stream type:134
logger.js:40 [log] > Parsed video,PTS:[69.000,72.000],DTS:[68.967/71.967],nb:90,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,71.967]
logger.js:40 [log] > latency/loading/parsing/append/kbps:155/99/27/2/10766
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [798,818],duration:59.992000000000026
logger.js:40 [log] > live audio playlist sliding:12.012
logger.js:40 [log] > Loading 818, cc: 0 of [798 ,818],track 0, currentTime:62.674123,bufferEnd:69.016
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > Demuxing 818 of [798 ,818],track 0
logger.js:40 [log] > parsed audio,PTS:[69.016,72.003],DTS:[69.016/72.003],nb:140
logger.js:40 [log] > audio stream:PARSING->PARSED
logger.js:40 [log] > audio buffered : [0.001,3.009][51.011,72.003]
logger.js:40 [log] > audio stream:PARSED->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2760 ms
logger.js:40 [log] > level 0 loaded [799,819],duration:60
logger.js:40 [log] > live playlist sliding:15.000
logger.js:40 [log] > Updating Media Source duration to 75.000
logger.js:40 [log] > Loading 819 of [799 ,819],level 0, currentTime:64.605,bufferEnd:71.967
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 819 of [799 ,819],level 0
logger.js:40 [log] > Parsing 819 of [799 ,819],level 0, cc 0
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > Unknown stream type:134
logger.js:40 [log] > Parsed video,PTS:[72.000,75.000],DTS:[71.967/74.967],nb:90,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,74.967]
logger.js:40 [log] > latency/loading/parsing/append/kbps:122/114/17/3/13729
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [799,819],duration:59.99200000000002
logger.js:40 [log] > live audio playlist sliding:15.020
logger.js:40 [log] > Loading 819, cc: 0 of [799 ,819],track 0, currentTime:65.805471,bufferEnd:72.003
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > Demuxing 819 of [799 ,819],track 0
logger.js:40 [log] > parsed audio,PTS:[72.003,75.011],DTS:[72.003/75.011],nb:141
logger.js:40 [log] > audio stream:PARSING->PARSED
logger.js:40 [log] > audio buffered : [0.001,3.009][51.011,75.011]
logger.js:40 [log] > audio stream:PARSED->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2748 ms
logger.js:40 [log] > level 0 loaded [800,820],duration:60
logger.js:40 [log] > live playlist sliding:18.000
logger.js:40 [log] > Updating Media Source duration to 78.000
logger.js:40 [log] > Loading 820 of [800 ,820],level 0, currentTime:67.488,bufferEnd:74.967
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 820 of [800 ,820],level 0
logger.js:40 [log] > Parsing 820 of [800 ,820],level 0, cc 0
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > Unknown stream type:134
logger.js:40 [log] > Parsed video,PTS:[75.000,78.000],DTS:[74.967/77.967],nb:90,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,77.967]
logger.js:40 [log] > latency/loading/parsing/append/kbps:127/47/18/3/14757
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [800,820],duration:60.01300000000002
logger.js:40 [log] > live audio playlist sliding:18.007
logger.js:40 [log] > Loading 820, cc: 0 of [800 ,820],track 0, currentTime:68.933773,bufferEnd:75.011
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > Demuxing 820 of [800 ,820],track 0
logger.js:40 [log] > parsed audio,PTS:[75.011,78.019],DTS:[75.011/78.019],nb:141
logger.js:40 [log] > audio stream:PARSING->PARSED
logger.js:40 [log] > audio buffered : [0.001,3.009][51.011,78.019]
logger.js:40 [log] > audio stream:PARSED->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2773 ms
logger.js:40 [log] > level 0 loaded [801,821],duration:60
logger.js:40 [log] > live playlist sliding:21.000
logger.js:40 [log] > Updating Media Source duration to 81.000
logger.js:40 [log] > Loading 821 of [801 ,821],level 0, currentTime:70.340,bufferEnd:77.967
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 821 of [801 ,821],level 0
logger.js:40 [log] > Parsing 821 of [801 ,821],level 0, cc 0
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > Unknown stream type:134
logger.js:40 [log] > Parsed video,PTS:[78.000,81.000],DTS:[77.967/80.967],nb:90,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,80.967]
logger.js:40 [log] > latency/loading/parsing/append/kbps:108/43/20/3/16492
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [801,821],duration:59.99200000000002
logger.js:40 [log] > live audio playlist sliding:21.015
logger.js:40 [log] > Loading 821, cc: 0 of [801 ,821],track 0, currentTime:72.278381,bufferEnd:78.019
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > Demuxing 821 of [801 ,821],track 0
logger.js:40 [log] > parsed audio,PTS:[78.019,81.006],DTS:[78.019/81.006],nb:140
logger.js:40 [log] > audio stream:PARSING->PARSED
logger.js:40 [log] > audio buffered : [0.001,3.009][51.011,81.006]
logger.js:40 [log] > audio stream:PARSED->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2772 ms
logger.js:40 [log] > level 0 loaded [802,822],duration:60
logger.js:40 [log] > live playlist sliding:24.000
logger.js:40 [log] > Updating Media Source duration to 84.000
logger.js:40 [log] > Loading 822 of [802 ,822],level 0, currentTime:73.215,bufferEnd:80.967
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 822 of [802 ,822],level 0
logger.js:40 [log] > Parsing 822 of [802 ,822],level 0, cc 0
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > Unknown stream type:134
logger.js:40 [log] > Parsed video,PTS:[81.000,84.000],DTS:[80.967/83.967],nb:90,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,83.967]
logger.js:40 [log] > latency/loading/parsing/append/kbps:118/73/21/10/14536
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [802,822],duration:60.01300000000002
logger.js:40 [log] > live audio playlist sliding:24.002
logger.js:40 [log] > Loading 822, cc: 0 of [802 ,822],track 0, currentTime:75.395403,bufferEnd:81.006
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > Demuxing 822 of [802 ,822],track 0
logger.js:40 [log] > parsed audio,PTS:[81.006,84.014],DTS:[81.006/84.014],nb:141
logger.js:40 [log] > audio stream:PARSING->PARSED
logger.js:40 [log] > audio buffered : [0.001,3.009][51.011,84.014]
logger.js:40 [log] > audio stream:PARSED->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2772 ms
logger.js:40 [log] > level 0 loaded [803,823],duration:60
logger.js:40 [log] > live playlist sliding:27.000
logger.js:40 [log] > Updating Media Source duration to 87.000
logger.js:40 [log] > Loading 823 of [803 ,823],level 0, currentTime:76.089,bufferEnd:83.967
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 823 of [803 ,823],level 0
logger.js:40 [log] > Parsing 823 of [803 ,823],level 0, cc 0
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > Unknown stream type:134
logger.js:40 [log] > Parsed video,PTS:[84.000,87.000],DTS:[83.967/86.967],nb:90,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,86.967]
logger.js:40 [log] > latency/loading/parsing/append/kbps:113/73/27/3/16293
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [804,824],duration:59.99200000000002
logger.js:40 [log] > live audio playlist sliding:30.018
logger.js:40 [log] > Loading 823, cc: 0 of [804 ,824],track 0, currentTime:78.522355,bufferEnd:84.014
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > Demuxing 823 of [804 ,824],track 0
logger.js:40 [log] > parsed audio,PTS:[84.014,87.022],DTS:[84.014/87.022],nb:141
logger.js:40 [log] > audio stream:PARSING->PARSED
logger.js:40 [log] > audio buffered : [0.001,3.009][51.011,87.022]
logger.js:40 [log] > audio stream:PARSED->IDLE
logger.js:40 [log] > Loading 824, cc: 0 of [804 ,824],track 0, currentTime:78.753349,bufferEnd:87.022
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > Demuxing 824 of [804 ,824],track 0
logger.js:40 [log] > parsed audio,PTS:[87.022,90.008],DTS:[87.022/90.008],nb:140
logger.js:40 [log] > audio stream:PARSING->PARSED
logger.js:40 [log] > audio buffered : [0.001,3.009][51.011,90.008]
logger.js:40 [log] > audio stream:PARSED->IDLE
logger.js:40 [log] > live playlist, reload in 2779 ms
logger.js:40 [log] > level 0 loaded [804,824],duration:60
logger.js:40 [log] > live playlist sliding:30.000
logger.js:40 [log] > Loading 824 of [804 ,824],level 0, currentTime:78.955,bufferEnd:86.967
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 824 of [804 ,824],level 0
logger.js:40 [log] > Parsing 824 of [804 ,824],level 0, cc 0
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > Unknown stream type:134
logger.js:40 [log] > Parsed video,PTS:[87.000,90.000],DTS:[86.967/89.967],nb:90,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,89.967]
logger.js:40 [log] > latency/loading/parsing/append/kbps:105/108/20/4/13088
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [804,824],duration:59.99200000000002
logger.js:40 [log] > live audio playlist sliding:30.018
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 1429 ms
logger.js:40 [log] > level 0 loaded [804,824],duration:60
logger.js:40 [log] > live playlist sliding:30.000
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2753 ms
logger.js:40 [log] > level 0 loaded [805,825],duration:60
logger.js:40 [log] > live playlist sliding:33.000
logger.js:40 [log] > Updating Media Source duration to 93.000
logger.js:40 [log] > Loading 825 of [805 ,825],level 0, currentTime:83.399,bufferEnd:89.967
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 825 of [805 ,825],level 0
logger.js:40 [log] > Parsing 825 of [805 ,825],level 0, cc 0
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > Unknown stream type:134
logger.js:40 [log] > Parsed video,PTS:[90.000,93.000],DTS:[89.967/92.967],nb:90,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,92.967]
logger.js:40 [log] > latency/loading/parsing/append/kbps:99/74/20/4/18594
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [805,825],duration:60.01300000000002
logger.js:40 [log] > live audio playlist sliding:33.005
logger.js:40 [log] > Loading 825, cc: 0 of [805 ,825],track 0, currentTime:84.702886,bufferEnd:90.008
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > Demuxing 825 of [805 ,825],track 0
logger.js:40 [log] > parsed audio,PTS:[90.008,93.016],DTS:[90.008/93.016],nb:141
logger.js:40 [log] > audio stream:PARSING->PARSED
logger.js:40 [log] > audio buffered : [0.001,3.009][51.011,93.016]
logger.js:40 [log] > audio stream:PARSED->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2762 ms
logger.js:40 [log] > level 0 loaded [806,826],duration:60
logger.js:40 [log] > live playlist sliding:36.000
logger.js:40 [log] > Updating Media Source duration to 96.000
logger.js:40 [log] > Loading 826 of [806 ,826],level 0, currentTime:86.266,bufferEnd:92.967
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 826 of [806 ,826],level 0
logger.js:40 [log] > Parsing 826 of [806 ,826],level 0, cc 0
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > Unknown stream type:134
logger.js:40 [log] > Parsed video,PTS:[93.000,96.000],DTS:[92.967/95.967],nb:90,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,95.967]
logger.js:40 [log] > latency/loading/parsing/append/kbps:117/150/24/4/10619
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [806,826],duration:59.992000000000026
logger.js:40 [log] > live audio playlist sliding:36.013
logger.js:40 [log] > Loading 826, cc: 0 of [806 ,826],track 0, currentTime:87.821769,bufferEnd:93.016
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > Demuxing 826 of [806 ,826],track 0
logger.js:40 [log] > parsed audio,PTS:[93.016,96.003],DTS:[93.016/96.003],nb:140
logger.js:40 [log] > audio stream:PARSING->PARSED
logger.js:40 [log] > audio buffered : [0.001,3.009][51.011,96.003]
logger.js:40 [log] > audio stream:PARSED->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2772 ms
logger.js:40 [log] > level 0 loaded [807,827],duration:60
logger.js:40 [log] > live playlist sliding:39.000
logger.js:40 [log] > Updating Media Source duration to 99.000
logger.js:40 [log] > Loading 827 of [807 ,827],level 0, currentTime:89.138,bufferEnd:95.967
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 827 of [807 ,827],level 0
logger.js:40 [log] > Parsing 827 of [807 ,827],level 0, cc 0
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > Unknown stream type:134
logger.js:40 [log] > Parsed video,PTS:[96.000,99.000],DTS:[95.967/98.967],nb:90,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,98.967]
logger.js:40 [log] > latency/loading/parsing/append/kbps:118/121/21/11/14496
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [809,829],duration:57.83700000000002
logger.js:40 [log] > live audio playlist sliding:45.016
logger.js:40 [log] > Loading 827, cc: 0 of [809 ,829],track 0, currentTime:90.961929,bufferEnd:96.003
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > Demuxing 827 of [809 ,829],track 0
logger.js:40 [log] > parsed audio,PTS:[96.003,99.011],DTS:[96.003/99.011],nb:141
logger.js:40 [log] > audio stream:PARSING->PARSED
logger.js:40 [log] > audio buffered : [0.001,3.009][51.011,99.011]
logger.js:40 [log] > audio stream:PARSED->IDLE
logger.js:40 [log] > Loading 828, cc: 0 of [809 ,829],track 0, currentTime:91.096442,bufferEnd:99.011
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > Demuxing 828 of [809 ,829],track 0
logger.js:40 [log] > parsed audio,PTS:[99.011,102.019],DTS:[99.011/102.019],nb:141
logger.js:40 [log] > audio stream:PARSING->PARSED
logger.js:40 [log] > audio buffered : [0.001,3.009][51.011,102.019]
logger.js:40 [log] > audio stream:PARSED->IDLE
logger.js:40 [log] > Loading 829, cc: 0 of [809 ,829],track 0, currentTime:91.287341,bufferEnd:102.019
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > Demuxing 829 of [809 ,829],track 0
logger.js:40 [log] > parsed audio,PTS:[102.019,102.851],DTS:[102.019/102.851],nb:39
logger.js:40 [log] > audio stream:PARSING->PARSED
logger.js:40 [log] > audio buffered : [0.001,3.009][51.011,102.851]
logger.js:40 [log] > audio stream:PARSED->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2677 ms
logger.js:40 [log] > level 0 loaded [809,829],duration:57.833
logger.js:40 [log] > live playlist sliding:45.000
logger.js:40 [log] > Loading 828 of [809 ,829],level 0, currentTime:92.014,bufferEnd:98.967
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 828 of [809 ,829],level 0
logger.js:40 [log] > Parsing 828 of [809 ,829],level 0, cc 0
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > Unknown stream type:134
logger.js:40 [log] > Parsed video,PTS:[99.000,102.000],DTS:[98.967/101.967],nb:90,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,101.967]
logger.js:40 [log] > latency/loading/parsing/append/kbps:109/43/31/3/14535
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > Loading 829 of [809 ,829],level 0, currentTime:92.209,bufferEnd:101.967
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 829 of [809 ,829],level 0
logger.js:40 [log] > Parsing 829 of [809 ,829],level 0, cc 0
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > Unknown stream type:134
logger.js:40 [log] > Parsed video,PTS:[102.000,102.833],DTS:[101.967/102.800],nb:25,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,102.800]
logger.js:40 [log] > latency/loading/parsing/append/kbps:114/12/5/2/10194
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [809,831],duration:61.88999990000002
logger.js:40 [log] > live audio playlist sliding:45.016
logger.js:40 [log] > Loading 830, cc: 1 of [809 ,831],track 0, currentTime:94.147749,bufferEnd:102.851
logger.js:40 [log] > audio stream:IDLE->FRAG_LOADING
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > audio stream:FRAG_LOADING->PARSING
logger.js:40 [log] > unknown video PTS for continuity counter 1, waiting for video PTS before demuxing audio frag 830 of [809 ,831],track 0
logger.js:40 [log] > audio stream:PARSING->WAITING_INIT_PTS
logger.js:40 [warn] > Waiting fragment CC (1) does not match video track CC (0)
(anonymous) @ logger.js:40
doTick @ audio-stream-controller.js:335
tick @ task-loop.ts:111
setInterval (async)
setInterval @ task-loop.ts:73
onAudioTrackSwitching @ audio-stream-controller.js:412
onEventGeneric @ event-handler.ts:82
onEvent @ event-handler.ts:69
emit @ index.js:203
trigger @ observer.ts:13
_setAudioTrack @ audio-track-controller.js:233
(anonymous) @ audio-track-controller.js:303
traverseTracks @ audio-track-controller.js:293
_selectInitialAudioTrack @ audio-track-controller.js:309
_selectAudioGroup @ audio-track-controller.js:258
onManifestParsed @ audio-track-controller.js:95
onEventGeneric @ event-handler.ts:82
onEvent @ event-handler.ts:69
emit @ index.js:203
trigger @ observer.ts:13
onManifestLoaded @ level-controller.js:150
onEventGeneric @ event-handler.ts:82
onEvent @ event-handler.ts:69
emit @ index.js:203
trigger @ observer.ts:13
_handleMasterPlaylist @ playlist-loader.ts:330
loadsuccess @ playlist-loader.ts:267
readystatechange @ xhr-loader.js:118
XMLHttpRequest.send (async)
(anonymous) @ VM93:1
loadInternal @ xhr-loader.js:80
load @ xhr-loader.js:38
load @ playlist-loader.ts:235
onManifestLoading @ playlist-loader.ts:132
onEventGeneric @ event-handler.ts:82
onEvent @ event-handler.ts:69
emit @ index.js:203
trigger @ observer.ts:13
loadSource @ hls.ts:288
loadSelectedStream @ main.js:249
applyConfigEditorValue @ main.js:1377
onclick @ (index):106
logger.js:40 [log] > audio stream:WAITING_INIT_PTS->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2602 ms
logger.js:40 [log] > level 0 loaded [809,831],duration:61.833
logger.js:40 [log] > live playlist sliding:45.000
logger.js:40 [log] > Updating Media Source duration to 106.833
logger.js:40 [log] > Loading 830 of [809 ,831],level 0, currentTime:94.806,bufferEnd:102.800
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 830 of [809 ,831],level 0
logger.js:40 [log] > Parsing 830 of [809 ,831],level 0, cc 1
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
logger.js:40 [log] > main:discontinuity detected
logger.js:40 [log] > InitPTS for cc: 1 found from video track: -9072000
logger.js:40 [log] > main track:video,container:video/mp4,codecs[level/parsed]=[avc1.4D401F/avc1.64001f]
logger.js:40 [log] > Parsed video,PTS:[102.833,104.833],DTS:[102.800/104.800],nb:60,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,104.800]
logger.js:40 [log] > latency/loading/parsing/append/kbps:92/148/49/3/8683
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > Loading 831 of [809 ,831],level 0, currentTime:95.110,bufferEnd:104.800
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 831 of [809 ,831],level 0
logger.js:40 [log] > Parsing 831 of [809 ,831],level 0, cc 1
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
logger.js:40 [log] > Parsed video,PTS:[104.833,106.833],DTS:[104.800/106.800],nb:60,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,106.800]
logger.js:40 [log] > latency/loading/parsing/append/kbps:86/59/13/3/18704
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [810,832],duration:60.88699990000002
logger.js:40 [log] > live audio playlist sliding:48.003
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2548 ms
logger.js:40 [log] > level 0 loaded [810,832],duration:60.833
logger.js:40 [log] > live playlist sliding:48.000
logger.js:40 [log] > Updating Media Source duration to 108.833
logger.js:40 [log] > Loading 832 of [810 ,832],level 0, currentTime:97.522,bufferEnd:106.800
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 832 of [810 ,832],level 0
logger.js:40 [log] > Parsing 832 of [810 ,832],level 0, cc 1
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
logger.js:40 [log] > Parsed video,PTS:[106.833,108.833],DTS:[106.800/108.800],nb:60,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,108.800]
logger.js:40 [log] > latency/loading/parsing/append/kbps:79/37/26/4/18398
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2580 ms
logger.js:40 [log] > level 0 loaded [812,834],duration:61.833
logger.js:40 [log] > live playlist sliding:51.000
logger.js:40 [log] > Updating Media Source duration to 112.833
logger.js:40 [log] > Loading 833 of [812 ,834],level 0, currentTime:100.203,bufferEnd:108.800
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > Loaded 833 of [812 ,834],level 0
logger.js:40 [log] > Parsing 833 of [812 ,834],level 0, cc 1
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
logger.js:40 [log] > Parsed video,PTS:[108.833,110.833],DTS:[108.800/110.800],nb:60,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,110.800]
logger.js:40 [log] > latency/loading/parsing/append/kbps:122/7/50/9/16241
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > Loading 834 of [812 ,834],level 0, currentTime:100.406,bufferEnd:110.800
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > track 0 loaded [812,834],duration:61.88899970000003
logger.js:40 [log] > live audio playlist sliding:51.011
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > Loaded 834 of [812 ,834],level 0
logger.js:40 [log] > Parsing 834 of [812 ,834],level 0, cc 1
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
logger.js:40 [log] > Parsed video,PTS:[110.833,112.833],DTS:[110.800/112.800],nb:60,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,112.800]
logger.js:40 [log] > latency/loading/parsing/append/kbps:97/64/9/2/16525
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 1344 ms
logger.js:40 [log] > level 0 loaded [812,834],duration:61.833
logger.js:40 [log] > live playlist sliding:51.000
logger.js:40 [warn] > Playback stalling at @102.801696 due to low buffer (buffer=0.04930299999999477)
(anonymous) @ logger.js:40
_reportStall @ gap-controller.js:161
poll @ gap-controller.js:107
_checkBuffer @ stream-controller.js:1296
doTick @ stream-controller.js:136
tick @ task-loop.ts:111
setInterval (async)
setInterval @ task-loop.ts:73
startLoad @ stream-controller.js:60
(anonymous) @ hls.ts:301
startLoad @ hls.ts:300
onManifestParsed @ stream-controller.js:767
onEventGeneric @ event-handler.ts:82
onEvent @ event-handler.ts:69
emit @ index.js:203
trigger @ observer.ts:13
onManifestLoaded @ level-controller.js:150
onEventGeneric @ event-handler.ts:82
onEvent @ event-handler.ts:69
emit @ index.js:203
trigger @ observer.ts:13
_handleMasterPlaylist @ playlist-loader.ts:330
loadsuccess @ playlist-loader.ts:267
readystatechange @ xhr-loader.js:118
XMLHttpRequest.send (async)
(anonymous) @ VM93:1
loadInternal @ xhr-loader.js:80
load @ xhr-loader.js:38
load @ playlist-loader.ts:235
onManifestLoading @ playlist-loader.ts:132
onEventGeneric @ event-handler.ts:82
onEvent @ event-handler.ts:69
emit @ index.js:203
trigger @ observer.ts:13
loadSource @ hls.ts:288
loadSelectedStream @ main.js:249
applyConfigEditorValue @ main.js:1377
onclick @ (index):106
main.js:565 Error event: {type: "mediaError", details: "bufferStalledError", fatal: false, buffer: 0.04930299999999477}
(anonymous) @ main.js:565
emit @ index.js:203
trigger @ observer.ts:13
_reportStall @ gap-controller.js:162
poll @ gap-controller.js:107
_checkBuffer @ stream-controller.js:1296
doTick @ stream-controller.js:136
tick @ task-loop.ts:111
setInterval (async)
setInterval @ task-loop.ts:73
startLoad @ stream-controller.js:60
(anonymous) @ hls.ts:301
startLoad @ hls.ts:300
onManifestParsed @ stream-controller.js:767
onEventGeneric @ event-handler.ts:82
onEvent @ event-handler.ts:69
emit @ index.js:203
trigger @ observer.ts:13
onManifestLoaded @ level-controller.js:150
onEventGeneric @ event-handler.ts:82
onEvent @ event-handler.ts:69
emit @ index.js:203
trigger @ observer.ts:13
_handleMasterPlaylist @ playlist-loader.ts:330
loadsuccess @ playlist-loader.ts:267
readystatechange @ xhr-loader.js:118
XMLHttpRequest.send (async)
(anonymous) @ VM93:1
loadInternal @ xhr-loader.js:80
load @ xhr-loader.js:38
load @ playlist-loader.ts:235
onManifestLoading @ playlist-loader.ts:132
onEventGeneric @ event-handler.ts:82
onEvent @ event-handler.ts:69
emit @ index.js:203
trigger @ observer.ts:13
loadSource @ hls.ts:288
loadSelectedStream @ main.js:249
applyConfigEditorValue @ main.js:1377
onclick @ (index):106
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [812,834],duration:61.88899970000003
logger.js:40 [log] > live audio playlist sliding:51.011
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2538 ms
logger.js:40 [log] > level 0 loaded [813,835],duration:60.833
logger.js:40 [log] > live playlist sliding:54.000
logger.js:40 [log] > Updating Media Source duration to 114.833
logger.js:40 [log] > Loading 835 of [813 ,835],level 0, currentTime:102.802,bufferEnd:112.800
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 835 of [813 ,835],level 0
logger.js:40 [log] > Parsing 835 of [813 ,835],level 0, cc 1
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
logger.js:40 [log] > Parsed video,PTS:[112.833,114.833],DTS:[112.800/114.800],nb:60,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,114.800]
logger.js:40 [log] > latency/loading/parsing/append/kbps:88/43/21/3/17421
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [814,836],duration:60.82199960000002
logger.js:40 [log] > live audio playlist sliding:57.006
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2542 ms
logger.js:40 [log] > level 0 loaded [814,836],duration:60.8
logger.js:40 [log] > live playlist sliding:57.000
logger.js:40 [log] > Updating Media Source duration to 117.800
logger.js:40 [log] > Loading 836 of [814 ,836],level 0, currentTime:102.802,bufferEnd:114.800
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 836 of [814 ,836],level 0
logger.js:40 [log] > Parsing 836 of [814 ,836],level 0, cc 1
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
blob:https://hls-js.netlify.app/f2c9914d-5f66-4ba8-ad85-32b3a747660d:562 [log] > 1524000/1524000:unknown NAL 11 
logger.js:40 [log] > Parsed video,PTS:[114.833,117.800],DTS:[114.800/117.767],nb:89,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,117.767]
logger.js:40 [log] > latency/loading/parsing/append/kbps:95/53/22/7/24499
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > live playlist, reload in 2459 ms
logger.js:40 [log] > level 0 loaded [815,838],duration:61.8
logger.js:40 [log] > live playlist sliding:60.000
logger.js:40 [log] > Updating Media Source duration to 121.800
logger.js:40 [log] > Loading 837 of [815 ,838],level 0, currentTime:102.802,bufferEnd:117.767
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > track 0 loaded [815,838],duration:61.86699950000003
logger.js:40 [log] > live audio playlist sliding:60.014
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > Loaded 837 of [815 ,838],level 0
logger.js:40 [log] > Parsing 837 of [815 ,838],level 0, cc 2
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
logger.js:40 [log] > main:discontinuity detected
logger.js:40 [log] > InitPTS for cc: 2 found from video track: -10419000
logger.js:40 [log] > main track:video,container:video/mp4,codecs[level/parsed]=[avc1.4D401F/avc1.64001f]
logger.js:40 [log] > Parsed video,PTS:[117.800,119.800],DTS:[117.767/119.767],nb:60,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,119.767]
logger.js:40 [log] > latency/loading/parsing/append/kbps:91/86/34/10/11909
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > Loading 838 of [815 ,838],level 0, currentTime:102.802,bufferEnd:119.767
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 838 of [815 ,838],level 0
logger.js:40 [log] > Parsing 838 of [815 ,838],level 0, cc 2
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
logger.js:40 [log] > Parsed video,PTS:[119.800,121.800],DTS:[119.767/121.767],nb:60,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,121.767]
logger.js:40 [log] > latency/loading/parsing/append/kbps:83/51/11/5/19918
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2438 ms
logger.js:40 [log] > level 0 loaded [816,839],duration:60.8
logger.js:40 [log] > live playlist sliding:63.000
logger.js:40 [log] > Updating Media Source duration to 123.800
logger.js:40 [log] > Loading 839 of [816 ,839],level 0, currentTime:102.802,bufferEnd:121.767
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 839 of [816 ,839],level 0
logger.js:40 [log] > Parsing 839 of [816 ,839],level 0, cc 2
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
logger.js:40 [log] > Parsed video,PTS:[121.800,123.800],DTS:[121.767/123.767],nb:60,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,123.767]
logger.js:40 [log] > latency/loading/parsing/append/kbps:87/40/22/3/17553
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [816,839],duration:60.84299950000002
logger.js:40 [log] > live audio playlist sliding:63.022
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2348 ms
logger.js:40 [log] > level 0 loaded [817,841],duration:61.8
logger.js:40 [log] > live playlist sliding:66.000
logger.js:40 [log] > Updating Media Source duration to 127.800
logger.js:40 [log] > Loading 840 of [817 ,841],level 0, currentTime:102.802,bufferEnd:123.767
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 840 of [817 ,841],level 0
logger.js:40 [log] > Parsing 840 of [817 ,841],level 0, cc 2
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
logger.js:40 [log] > Parsed video,PTS:[123.800,125.800],DTS:[123.767/125.767],nb:60,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,125.767]
logger.js:40 [log] > latency/loading/parsing/append/kbps:144/124/26/3/9749
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > Loading 841 of [817 ,841],level 0, currentTime:102.802,bufferEnd:125.767
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 841 of [817 ,841],level 0
logger.js:40 [log] > Parsing 841 of [817 ,841],level 0, cc 2
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
logger.js:40 [log] > Parsed video,PTS:[125.800,127.800],DTS:[125.767/127.767],nb:60,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,127.767]
logger.js:40 [log] > latency/loading/parsing/append/kbps:151/151/24/26/8392
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [817,841],duration:61.86599930000002
logger.js:40 [log] > live audio playlist sliding:66.008
logger.js:40 [log] > audioTrack 0 loaded
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 1236 ms
logger.js:40 [log] > level 0 loaded [817,841],duration:61.8
logger.js:40 [log] > live playlist sliding:66.000
logger.js:40 [log] > Attempt loading level index 0 with URL-id 0
logger.js:40 [log] > live playlist, reload in 2329 ms
logger.js:40 [log] > level 0 loaded [818,842],duration:60.8
logger.js:40 [log] > live playlist sliding:69.000
logger.js:40 [log] > Updating Media Source duration to 129.800
logger.js:40 [log] > Loading 842 of [818 ,842],level 0, currentTime:102.802,bufferEnd:127.767
logger.js:40 [log] > main stream-controller: IDLE->FRAG_LOADING
logger.js:40 [log] > Loaded 842 of [818 ,842],level 0
logger.js:40 [log] > Parsing 842 of [818 ,842],level 0, cc 2
logger.js:40 [log] > main stream-controller: FRAG_LOADING->PARSING
logger.js:40 [log] > Parsed video,PTS:[127.800,129.800],DTS:[127.767/129.767],nb:60,dropped:0
logger.js:40 [log] > main stream-controller: PARSING->PARSED
logger.js:40 [log] > main buffered : [50.967,129.767]
logger.js:40 [log] > latency/loading/parsing/append/kbps:76/35/17/2/21910
logger.js:40 [log] > main stream-controller: PARSED->IDLE
logger.js:40 [log] > trying to update audio-track 0
logger.js:40 [log] > loading audio-track playlist for id: 0
logger.js:40 [log] > track 0 loaded [818,842],duration:60.86299920000002
logger.js:40 [log] > live audio playlist sliding:69.016
logger.js:40 [log] > audioTrack 0 loaded
```

```
For media errors reported on Chrome browser, please also paste the output of chrome://media-internals 
```

--- a/src/controller/audio-stream-controller.js
+++ b/src/controller/audio-stream-controller.js
@@ -47,6 +47,7 @@ class AudioStreamController extends BaseStreamController {
     this.initPTS = [];
     this.waitingFragment = null;
     this.videoTrackCC = null;
+    this.waitingVideoCC = null;
   }
 
   // Signal that video PTS was found
@@ -305,13 +306,15 @@ class AudioStreamController extends BaseStreamController {
           this.waitingFragment = null;
           this.state = State.FRAG_LOADING;
           this.onFragLoaded(waitingFrag);
-        } else if (!this.fragPrevious) {
-          logger.log(`Waiting fragment cc (${waitingFragCC}) cancelled because of continuity change`);
+        } else if (this.videoTrackCC !== this.waitingVideoCC) {
+          // Drop waiting fragment if videoTrackCC has changed since waitingFragment was set and initPTS was not found
+          logger.log(`Waiting fragment cc (${waitingFragCC}) cancelled because video is at cc ${this.videoTrackCC}`);
           this.clearWaitingFragment();
         } else {
+          // Drop waiting fragment if an earlier fragment is needed
           const bufferInfo = BufferHelper.bufferInfo(this.mediaBuffer, this.media.currentTime, config.maxBufferHole);
           const waitingFragmentAtPosition = fragmentWithinToleranceTest(bufferInfo.end, config.maxFragLookUpTolerance, waitingFrag.frag);
-          if (waitingFragmentAtPosition !== 0) {
+          if (waitingFragmentAtPosition < 0) {
             logger.log(`Waiting fragment cc (${waitingFragCC}) @ ${waitingFrag.frag.start} cancelled because another fragment at ${bufferInfo.end} is needed`);
             this.clearWaitingFragment();
           }
@@ -337,6 +340,7 @@ class AudioStreamController extends BaseStreamController {
     if (waitingFrag) {
       this.fragmentTracker.removeFragment(waitingFrag.frag);
       this.waitingFragment = null;
+      this.waitingVideoCC = null;
       this.state = State.IDLE;
     }
   }
@@ -516,6 +520,7 @@ class AudioStreamController extends BaseStreamController {
         } else {
           logger.log(`Unknown video PTS for cc ${cc}, waiting for video PTS before demuxing audio frag ${sn} of [${details.startSN} ,${details.endSN}],track ${trackId}`);
           this.waitingFragment = data;
+          this.waitingVideoCC = this.videoTrackCC;
           this.state = State.WAITING_INIT_PTS;
         }
       }

--- a/src/controller/audio-stream-controller.js
+++ b/src/controller/audio-stream-controller.js
@@ -15,6 +15,8 @@ import { FragmentState } from './fragment-tracker';
 import { ElementaryStreamTypes } from '../loader/fragment';
 import BaseStreamController, { State } from './base-stream-controller';
 import { MAX_START_GAP_JUMP } from './gap-controller';
+import { fragmentWithinToleranceTest } from './fragment-finders';
+
 const { performance } = window;
 
 const TICK_INTERVAL = 100; // how often to tick in ms
@@ -218,31 +220,6 @@ class AudioStreamController extends BaseStreamController {
           let foundFrag;
           let maxFragLookUpTolerance = config.maxFragLookUpTolerance;
           const fragNext = fragPrevious ? fragments[fragPrevious.sn - fragments[0].sn + 1] : undefined;
-          let fragmentWithinToleranceTest = (candidate) => {
-            // offset should be within fragment boundary - config.maxFragLookUpTolerance
-            // this is to cope with situations like
-            // bufferEnd = 9.991
-            // frag[Ø] : [0,10]
-            // frag[1] : [10,20]
-            // bufferEnd is within frag[0] range ... although what we are expecting is to return frag[1] here
-            //              frag start               frag start+duration
-            //                  |-----------------------------|
-            //              <--->                         <--->
-            //  ...--------><-----------------------------><---------....
-            // previous frag         matching fragment         next frag
-            //  return -1             return 0                 return 1
-            // logger.log(`level/sn/start/end/bufEnd:${level}/${candidate.sn}/${candidate.start}/${(candidate.start+candidate.duration)}/${bufferEnd}`);
-            // Set the lookup tolerance to be small enough to detect the current segment - ensures we don't skip over very small segments
-            let candidateLookupTolerance = Math.min(maxFragLookUpTolerance, candidate.duration);
-            if ((candidate.start + candidate.duration - candidateLookupTolerance) <= bufferEnd) {
-              return 1;
-            } else if (candidate.start - candidateLookupTolerance > bufferEnd && candidate.start) {
-              // if maxFragLookUpTolerance will have negative value then don't return -1 for first element
-              return -1;
-            }
-
-            return 0;
-          };
 
           if (bufferEnd < end) {
             if (bufferEnd > end - maxFragLookUpTolerance) {
@@ -250,10 +227,10 @@ class AudioStreamController extends BaseStreamController {
             }
 
             // Prefer the next fragment if it's within tolerance
-            if (fragNext && !fragmentWithinToleranceTest(fragNext)) {
+            if (fragNext && !fragmentWithinToleranceTest(bufferEnd, maxFragLookUpTolerance, fragNext)) {
               foundFrag = fragNext;
             } else {
-              foundFrag = BinarySearch.search(fragments, fragmentWithinToleranceTest);
+              foundFrag = BinarySearch.search(fragments, (frag) => fragmentWithinToleranceTest(bufferEnd, maxFragLookUpTolerance, frag));
             }
           } else {
             // reach end of playlist
@@ -320,26 +297,24 @@ class AudioStreamController extends BaseStreamController {
       }
       break;
     case State.WAITING_INIT_PTS:
-      const videoTrackCC = this.videoTrackCC;
-      if (this.initPTS[videoTrackCC] === undefined) {
-        break;
-      }
-
       // Ensure we don't get stuck in the WAITING_INIT_PTS state if the waiting frag CC doesn't match any initPTS
       const waitingFrag = this.waitingFragment;
       if (waitingFrag) {
         const waitingFragCC = waitingFrag.frag.cc;
-        if (videoTrackCC !== waitingFragCC) {
-          track = this.tracks[this.trackId];
-          if (track.details && track.details.live) {
-            logger.warn(`Waiting fragment CC (${waitingFragCC}) does not match video track CC (${videoTrackCC})`);
-            this.waitingFragment = null;
-            this.state = State.IDLE;
-          }
-        } else {
-          this.state = State.FRAG_LOADING;
-          this.onFragLoaded(this.waitingFragment);
+        if (this.initPTS[waitingFragCC] !== undefined) {
           this.waitingFragment = null;
+          this.state = State.FRAG_LOADING;
+          this.onFragLoaded(waitingFrag);
+        } else if (!this.fragPrevious) {
+          logger.warn(`Waiting fragment CC (${waitingFragCC}) cancelled because of continuity change`);
+          this.clearWaitingFragment();
+        } else {
+          const bufferInfo = BufferHelper.bufferInfo(this.mediaBuffer, this.media.currentTime, config.maxBufferHole);
+          const waitingFragmentAtPosition = fragmentWithinToleranceTest(bufferInfo.end, config.maxFragLookUpTolerance, waitingFrag.frag);
+          if (waitingFragmentAtPosition !== 0) {
+            logger.warn(`Waiting fragment CC (${waitingFragCC}) @ ${waitingFrag.frag.start} cancelled because another fragment at ${bufferInfo.end} is needed`);
+            this.clearWaitingFragment();
+          }
         }
       } else {
         this.state = State.IDLE;
@@ -354,6 +329,15 @@ class AudioStreamController extends BaseStreamController {
       break;
     default:
       break;
+    }
+  }
+
+  clearWaitingFragment () {
+    const waitingFrag = this.waitingFragment;
+    if (waitingFrag) {
+      this.fragmentTracker.removeFragment(waitingFrag.frag);
+      this.waitingFragment = null;
+      this.state = State.IDLE;
     }
   }
 
@@ -399,8 +383,8 @@ class AudioStreamController extends BaseStreamController {
     this.trackId = data.id;
 
     this.fragCurrent = null;
+    this.clearWaitingFragment();
     this.state = State.PAUSED;
-    this.waitingFragment = null;
     // destroy useless demuxer when switching audio to main
     if (!altAudio) {
       if (this.demuxer) {

--- a/src/controller/audio-stream-controller.js
+++ b/src/controller/audio-stream-controller.js
@@ -179,7 +179,7 @@ class AudioStreamController extends BaseStreamController {
           end = fragments[fragLen - 1].start + fragments[fragLen - 1].duration,
           frag;
 
-          // When switching audio track, reload audio as close as possible to currentTime
+        // When switching audio track, reload audio as close as possible to currentTime
         if (audioSwitch) {
           if (trackDetails.live && !trackDetails.PTSKnown) {
             logger.log('switching audiotrack, live stream, unknown PTS,load first fragment');
@@ -306,13 +306,13 @@ class AudioStreamController extends BaseStreamController {
           this.state = State.FRAG_LOADING;
           this.onFragLoaded(waitingFrag);
         } else if (!this.fragPrevious) {
-          logger.warn(`Waiting fragment CC (${waitingFragCC}) cancelled because of continuity change`);
+          logger.log(`Waiting fragment cc (${waitingFragCC}) cancelled because of continuity change`);
           this.clearWaitingFragment();
         } else {
           const bufferInfo = BufferHelper.bufferInfo(this.mediaBuffer, this.media.currentTime, config.maxBufferHole);
           const waitingFragmentAtPosition = fragmentWithinToleranceTest(bufferInfo.end, config.maxFragLookUpTolerance, waitingFrag.frag);
           if (waitingFragmentAtPosition !== 0) {
-            logger.warn(`Waiting fragment CC (${waitingFragCC}) @ ${waitingFrag.frag.start} cancelled because another fragment at ${bufferInfo.end} is needed`);
+            logger.log(`Waiting fragment cc (${waitingFragCC}) @ ${waitingFrag.frag.start} cancelled because another fragment at ${bufferInfo.end} is needed`);
             this.clearWaitingFragment();
           }
         }
@@ -514,7 +514,7 @@ class AudioStreamController extends BaseStreamController {
           let accurateTimeOffset = false; // details.PTSKnown || !details.live;
           this.demuxer.push(data.payload, initSegmentData, audioCodec, null, fragCurrent, duration, accurateTimeOffset, initPTS);
         } else {
-          logger.log(`unknown video PTS for continuity counter ${cc}, waiting for video PTS before demuxing audio frag ${sn} of [${details.startSN} ,${details.endSN}],track ${trackId}`);
+          logger.log(`Unknown video PTS for cc ${cc}, waiting for video PTS before demuxing audio frag ${sn} of [${details.startSN} ,${details.endSN}],track ${trackId}`);
           this.waitingFragment = data;
           this.state = State.WAITING_INIT_PTS;
         }

